alchemy/test/cloudflare/bucket.test.ts

```diff
@@ -566,6 +567,98 @@ describe("R2 Bucket Resource", async () => {
       await scope.finalize();
     }
   });
+
+  test("bucket with custom domain", async (scope) => {
+    const bucketName = `${BRANCH_PREFIX.toLowerCase()}-test-custom-domain`;
+    const customDomain = "r2-test.samscode.me";
+    const zoneId = process.env.CLOUDFLARE_ZONE_ID;
```

Comment:
Should probably use the one used in Health Check, we definitely dont wanna skip those tests 🙂 
```suggestion
const ZONE_NAME = process.env.TEST_ZONE ?? process.env.ALCHEMY_TEST_DOMAIN!;

```